src/pages/Models.jsx

```diff
@@ -0,0 +1,18 @@
+import React, { useState, useContext } from 'react'
+import { Heading, Box, Grid, GridItem, Text } from '@chakra-ui/react'
+import Navbar from '../components/Navbar.jsx'
+import ModelBox from '../components/ModelBox.jsx'
+
+export default function Models() {
```

Comment:
for clarity's sake, maybe we could rename this function + file to 'ModelsShowcase' instead?

the current name of 'Models' means it is fairly easy to mix up with the 'models' used in the backend. no real risk of either one of these being called in the wrong context, but it's just a bit clearer to change this filename to something else.